frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx

```diff
@@ -0,0 +1,158 @@
+import { FC, PropsWithChildren } from 'react'
+
+import { Link as MuiLink } from '@mui/material'
+import { GetServerSideProps } from 'next'
+import { Trans, useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { NextSeo } from 'next-seo'
+
+import {
+  LearnPageLayout
+} from '../../components/LearnPageLayout'
+
+export interface ImportantCardProps extends PropsWithChildren { }
+const ImportantCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#ffe8a3]/[.3] p-5">{children}</p>
+)
+
+export interface DisclaimerCardProps extends PropsWithChildren { }
```

Comment:
Bit of a nitpick, but this doesn't have to be exported if it isn't used outside of this module.

Reply:
removed the export keyword